src/domain/service/note.ts

```diff
@@ -11,28 +12,45 @@ export default class NoteService {
    */
   public repository: NoteRepository;
 
+  public noteRelationsRepository: NoteRelationsRepository;
+
   /**
    * Note service constructor
    *
    * @param repository - note repository
+   * @param noteRelationsRepository - note relationship service
    */
-  constructor(repository: NoteRepository) {
+  constructor(repository: NoteRepository, noteRelationsRepository: NoteRelationsRepository) {
     this.repository = repository;
+    this.noteRelationsRepository = noteRelationsRepository;
```

Comment:
It seems to me that you need to change the `repository` to noteRepository to set him apart

Reply:
You mean just rename it to `noteRepository`?

Reply:
yes